src/common/cm/cm_load.cpp

```diff
@@ -771,7 +771,7 @@ void CM_LoadMap(Str::StringRef name)
 	std::error_code err;
 	std::string mapData = FS::PakPath::ReadFile(mapFile, err);
 	if (err) {
-		Sys::Drop("Could not load %s", mapFile.c_str());
+		Sys::Drop("Could not load %s: %s (code:%02x)", mapFile.c_str(), err.message(), err.value() );
```

Comment:
The use of an unprefixed hexadecimal number is misleading. If it had only digits 0-9, you couldn't tell it's supposed to be hexadecimal.

Reply:
Good point, fixed